composeApp/src/androidMain/sqldelight/tmdb/io/github/couchtracker/db/tmdb/MovieDetailsCache.sq

```diff
@@ -0,0 +1,23 @@
+import app.moviebase.tmdb.model.TmdbMovieDetail;
+import io.github.couchtracker.tmdb.TmdbLanguage;
+import kotlin.Int;
+
+CREATE TABLE MovieDetailsCache (
+  tmdbId INTEGER AS Int NOT NULL,
+  language TEXT AS TmdbLanguage NOT NULL,
+  details TEXT AS TmdbMovieDetail NOT NULL,
+  PRIMARY KEY(tmdbId, language)
+);
+
+get:
+SELECT details
+FROM MovieDetailsCache
+WHERE tmdbId = ? AND language = ?;
+
+put {
```

Comment:
Question: is there not a better way to do this? Perhaps something like this: https://www.sqlite.org/lang_upsert.html

Reply:
UPSERT was introduced in 3.24. See https://developer.android.com/reference/android/database/sqlite/package-summary

Reply:
Actually, let's use requery